src/lib/LocalMedia.ts

```diff
@@ -103,6 +105,17 @@ export default class LocalMedia extends TypedLocalMediaEventTarget {
         this.dispatchEvent(new CustomEvent("microphone_enabled", { detail: { enabled: newValue } }));
     }
 
+    async startScreenshare() {
+        const screenshareStream = await navigator.mediaDevices.getDisplayMedia();
```

Comment:
Maybe stop existing stream? Or just do a noop if stream already exists?

Reply:
Just returning existing stream if it exists now 👍